lib/internal/util/inspect.js

```diff
@@ -1131,10 +1133,30 @@ function formatProperty(ctx, value, recurseTimes, key, type) {
     }
     ctx.indentationLvl -= diff;
   } else if (desc.get !== undefined) {
-    if (desc.set !== undefined) {
-      str = ctx.stylize('[Getter/Setter]', 'special');
+    const gs = desc.set !== undefined ? 'Getter/Setter' : 'Getter';
```

Comment:
```suggestion
    const label = desc.set !== undefined ? 'Getter/Setter' : 'Getter';
```

or anything else less obscure